permissions/helpers.py

```diff
@@ -630,7 +535,4 @@ def get_service_sectors_where_can_view_reservations(user: AnyUser) -> models.Que
     if has_general_permission(user, general_permission):
         return ServiceSector.objects.all().values("pk")
 
-    service_sector_ids = [
-        pk for pk, perms in user.service_sector_permissions.items() if service_sector_permission.value in perms
-    ]
-    return ServiceSector.objects.filter(id__in=service_sector_ids).values("pk")
+    return ServiceSector.objects.none()
```

Comment:
Looks like `get_service_sectors_where_can_view_reservations` can be dropped altogether if it is removed also from RecurringReservationNode: https://github.com/City-of-Helsinki/tilavarauspalvelu-core/blob/b1a1e9d644ce26ee12911d04ca3405915f3061fc/api/graphql/types/recurring_reservation/types.py#L55

Reply:
Yes, done.